### PR TITLE
POST/DELETE requests

### PR DESCRIPTION
**Pull request by emmaev9**

### CODE REVIEW

File: TicketMS/Controllers/EventController.cs [hunk 1]
```diff
@@ -11,54 +12,76 @@ namespace TicketMS.Controllers
...
+
+        [HttpPost]
+        public ActionResult<EventAddDTO> Add(EventAddDTO eventDTO)
+        {
+            Event eventToSave = new Event();
+            eventToSave.Eventid = 7; 
+            //TO DO: Modify DB to have auto-incrment on primary keys
+            _mapper.Map(eventDTO,eventToSave);
+            _eventRepository.Add(eventToSave);
+            return NoContent();
```
Comment (claudiu-mera) [inline TicketMS/Controllers/EventController.cs]: Usually when we create a new entity, Created (201) response must be returned

File: TicketMS/Controllers/OrderController.cs [hunk 2]
```diff
@@ -11,49 +12,60 @@ namespace TicketMS.Controllers
...
-        public OrderController(IOrderRepository orderRepository)
+        public OrderController(IOrderRepository orderRepository, IMapper mapper)
         {
             _orderRepository = orderRepository;
+            _mapper = mapper;
         }
 
         [HttpGet]
-        public ActionResult<List<Order>> getAllOrders()
+        public async Task<ActionResult<List<Order>>> getAllOrders()
```
Comment (claudiu-mera) [inline TicketMS/Controllers/OrderController.cs]: All methods name in C# must start with uppercase letter

File: TicketMS/Repositories/Implementation/EventRepository.cs [hunk 3]
```diff
@@ -11,36 +12,35 @@ public EventRepository()
...
-        public IEnumerable<Event> GetAll()
+        public async Task<IEnumerable<Event>> GetAll()
         {
-            var events = _dbContext.Events;
+            var events = await _dbContext.Events.ToListAsync();
             return events;
         }
 
-        public Event GetById(int id)
+        public async Task<Event> GetById(int id)
```
Comment (claudiu-mera) [inline TicketMS/Repositories/Implementation/EventRepository.cs]: It is a good pratice to add Async sufix to asynchronous methods. For example, here it should be named GetByIdAsync.

Comment (claudiu-mera) [reply TicketMS/Repositories/Implementation/EventRepository.cs]: This is applicable for all async methods.

**Review by claudiu-mera — COMMENTED** [review]
You should also add a git ignore file, to not include dll, suo, or other files generated on local development machine. For example of gitignore content for a Visual Studio project, check: https://github.com/github/VisualStudio/blob/master/.gitignore

File: TicketMS/Controllers/OrderController.cs (hunk 2 above)
Comment (emmaev9) [reply TicketMS/Controllers/OrderController.cs]: Done

File: TicketMS/Repositories/Implementation/EventRepository.cs (hunk 3 above)
Comment (emmaev9) [reply TicketMS/Repositories/Implementation/EventRepository.cs]: Done

**Review by claudiu-mera — APPROVED** [review]

**Pull request merged**